### ISSUE TITLE
Enable features according to settings

### ISSUE THREAD

**Issue by Moddingear** [numworks/epsilon#1876]
TLDR : You've got an RGB LED, you can use it to signal differents versions of the firmware when in exam mode.

I'm an engineering student, and I've been supporting that calculator since it came out because of it's philosophy.
That philosophy has changed, and it's now basically "open-source" as in "you can look at our source code but can't do anything about it", and I don't like that. You've killed Omega and Delta, which had features that were really useful in the superior education, and you've nuked features because other countries don't allow it.

What I propose is that you enable features of the calculator according to settings, for example :

- When you're in a country like Portugal, and you've set you language to Portuguese, disable simplification and make the exam LED turn green
- When you're in a less restrictive country like France, enable the features allowed by the French governement and make the LED turn red
- And if you want to tinker with the source firmware and upload your own, make it like before but completely disable the LED, with an option to reset to the factory firmware in the settings (instead of whenever the calculator restarts)

You could even blink the colors of the country's flag...

But as of right now, I see this just like Apple did when people changed their battery and got a warning for doing it.
Your calculator is worse than the competition in terms of "being a calculator", and if you're worse than them in terms of modularity, I don't see the point anymore. 
In exams we're already scrutinised for having an odd-looking calculator, so I don't think that anything out of the ordinary will pass.

**Comment by Ecco** [comment]
I'm going to close this issue because it's insulting. I'm sorry, but no, our calculator is not worse than our competition in terms of being a calculator.

To answer some of your points though:
 - We already use a custom LED pattern whenever we are allowed to.
 - There's already an issue for the "reset" part at #1865

**Comment by LukasAppleFan** [comment]
Like at least give us the opportunity to downgrade and then use CFW, it's like Apple now, but Apple does this for security in some way, the thing is that I don't think we're in a time where a calculator could get hacked...
